### PR TITLE
[WFU] Fixes selected artist not showing when view is already presented

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

File: Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m [hunk 1]
```diff
@@ -477,6 +477,8 @@ - (void)presentRootViewController:(UIViewController *)viewController animated:(B
     
     // If there is an existing instance at that index, use it. Otherwise use the instance passed in as viewController.
     // If for some reason something went wrong, default to Home
+    BOOL alreadySelectedTab = self.selectedTabIndex == index;
```
Comment (l2succes) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: I moved that up so we can override the boolean based on the controller being presented

File: Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m [hunk 2]
```diff
@@ -485,12 +487,13 @@ - (void)presentRootViewController:(UIViewController *)viewController animated:(B
         case ARTopTabControllerIndexFavorites:
         case ARTopTabControllerIndexNotifications:
             presentableController = [[ARNavigationController alloc] initWithRootViewController:viewController];
+            alreadySelectedTab = NO;
```
Comment (l2succes) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: This makes sure we always present the new view controller instance

Comment (alloy) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: Can you add that as a comment to the code? Without it it's going to be very confusing to the next reader of the code.

Comment (l2succes) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: Sure

**Comment by alloy** [comment]
👍

**Pull request merged**